src/Microsoft.ML.LightGBM/Parallel/IParallel.cs

```diff
@@ -8,7 +8,7 @@
 using Microsoft.ML.EntryPoints;
 using Microsoft.ML.Runtime;
 
-namespace Microsoft.ML.LightGBM
+namespace Microsoft.ML.Trainers.LightGBM
```

Comment:
If we are changing all of these places, should we start using the new casing we decided on in #2762.

`namespace Microsoft.ML.Trainers.LightGbm`

Reply:
How about assembly names?

Reply:
It kind of feels like if that is our casing... we should probably case it the same way everywhere.